README.md

```diff
@@ -6,9 +6,9 @@
 
 ## Your bot list must:
 
-1. use a custom domain name with a respectable, paid TLD.
-2. not be a `[xyz].glitch.me` domain.
-   - Glitch is acceptable if:
+1. use a custom domain name with a respectable, paid TLD (not `.tk`, `.ga` or any free domain provider from freenom or any services like that provide domains for free. For more info read [here](https://www.quora.com/Is-Freenom-a-fraudulent-company-I-registered-free-ccTLD-ga-and-tk-domain-names-and-within-a-few-2-3-days-all-of-the-domains-were-taken-away-and-marked-as-Fraud-without-any-explanation) and the notice to freenom [here](https://www.internetnews.me/wp-content/uploads/2015/09/serad-to-zuurbier-23jun15-en.pdf) (PDF), also there are many more countless number of resources on google which point these services are fraud).
```

Comment:
```suggestion
1. use a custom domain name with a respectable, paid TLD.
```
I think listing specific examples here isn't needed and leads to folks going "mine isn't listed" + I don't think we need to have a load of links out giving context, folks should just be using a legitimate TLD, very simple.

Reply:
Agree. Pointing out tlds is (unintentionally) blaming them for offering a free tld, which is something to avoid imo.

Reply:
well, we need to blame them for giving free tlds. Freenom is **not** a free service. It takes away your domain once it has enough readers to come, and they redirect it to a dangerous ad service.

Reply:
BotBlock is not here to _blame_ or _attack_ services, please remove the additional content added to this line.

Reply:
No, we don't.
There is NEVER a point in blaming someone for anything that isn't illegal. Otherwise, could you blame other pages for similar behaviours? Like by your logic could you blame YouTube for offering free video uploading, since some use it to share porn or illegal stuff like terrorism.

Offering free TLDs isn't the best thing in the world, but in no way illegal, so just blatantly go like "This is baaaaaaaad!" is not good at all.

BotBlock should **not** have a bias towards specific things and pointing out specific TLDs as bad would make it biased.

Reply:
I get it now, I think this is already been resolved, u can stop blaming me now :weary: